### PR TITLE
added a notes page with sidebar

### PR DESCRIPTION
**Pull request by vidhi-thakur**
- added a notes page with sidebar
- refactored code 
- added custom css

### CODE REVIEW

File: src/components/sidebar/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+function Sidebar({ children }) {
+  return (
+    <div className='sidebar-HOC'>
+      <aside className='sidebar'>
+        {sidebarMenu.map(menu => <MenuItem
+          key={menu.name}
+          name={menu.name}
+          icon={menu.icon}
+          link={menu.link}
+        />)}
```
Comment (Sharmaryan) [inline src/components/sidebar/Sidebar.js:17]: why not to destructure from menu and then pass them or simple pass menu and destructure it inside the MenuItem component

**Review by Sharmaryan — APPROVED** [review]
suggested a small change, you can make it later also, it is not important but for readability it is good

**Review by sonishreyas — APPROVED** [review]
Code looks good. Realy like the folder structure

File: src/components/sidebar/Sidebar.js (hunk 1 above)
Comment (vidhi-thakur) [reply src/components/sidebar/Sidebar.js:17]: will update in the next iteration 👍

**Pull request merged**